### DISCUSSION TITLE
Database settings are reset after update

### DISCUSSION THREAD

**Discussion by DryestViper** [akiver/cs-demo-manager#677] [Q&A] [answered]
![Capture](https://github.com/akiver/cs-demo-manager/assets/153093332/4d7db185-9d1e-4fbe-8b2c-489e3ba1ac83)
I get this error after the most recent update anyone know a fix?

**Comment by emteka** [comment]
Did you follow these steps?
https://cs-demo-manager.com/docs/installation

I had same error but after removing all old postgres instances from my PC and reinstalling it - everything clicked

**Comment by DryestViper** [comment]
yes I installed and used the demo manager already and everything was working fine until I hit update in the top right corner. I will try reinstalling it now and get back to you

**Comment by DryestViper** [comment]
reinstalling everything fixed it thanks!

**Comment by DryestViper** [comment]
> reinstalling everything fixed it thanks!

**Comment by akiver** [comment, marked as answer]
@DryestViper the same problem will happen when upgrading from 3.0.1 to 3.0.2.

It has been tracked and fixed in https://github.com/akiver/cs-demo-manager/issues/671 🙏
